connector/github/github.go

```diff
@@ -350,17 +350,17 @@ func (c *githubConnector) listGroups(ctx context.Context, client *http.Client, u
 		// 'teams' list in config.
 		if len(org.Teams) == 0 {
 			inOrgNoTeams = true
-			c.logger.Debugf("github: user %q in org %q", userName, org.Name)
+			c.logger.Infof("github: user %q in org %q", userName, org.Name)
```

Comment:
I don't actually think we want to log this info. It's going to spam the logs and we'll already get the teams that a user is apart of based on other log messages